src/data/code-samples/en/accessibility-label/swift-ui.md

```diff
@@ -0,0 +1,31 @@
+# Accessibility label - SwiftUI
+
+In SwiftUI, the [`accessibilityLabel(_:)`](https://developer.apple.com/documentation/swiftui/view/accessibilitylabel(_:)-5f0zj) view modifier is used to assign an accessibility label to a view. It's essential to keep the accessibility label concise yet meaningful for optimal usability.
+
+```swift
+Link(destination: URL(string: "https://appt.org/en/")!, label: {
```

Comment:
Can you change this example to an `Image` with accessibilityLabel describing it.